packages/editor/src/extensions/task-list/task-list.ts

```diff
@@ -49,7 +53,15 @@ export const TaskListNode = TaskList.extend({
     return {
       stats: {
         default: { checked: 0, total: 0 },
-        rendered: false
+        rendered: false,
+        parseHTML: (element) => {
+          if (!element.dataset.title) return { checked: 0, total: 0 };
```

Comment:
What's this for?

Reply:
For nested task lists, the snapshots have `checked` and `total` counts as 0 regardless if they have task items. 

Without this guard clause, `checked` and `total` for nested tasks lists were updating

Reply:
That guard is incorrect in that case because even the top-most task list doesn't always have `data-title`. Let's instead do: `!!element.closest("ul")`. It's a bit inefficient but it'll be accurate.